yarn-endpoints-behavior.html

```diff
@@ -102,6 +102,15 @@
       },
 
       /**
+       * The encoded request path built from the computed URL
+       */
+      path: {
+        type: 'String',
+        readOnly: true,
```

Comment:
I thought setting readOnly would let the element itself change the prop, am I missing something here? Every test that touches the url fails because path can't be set since it's readOnly.